src/index.html

```diff
@@ -3,13 +3,115 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>HTML Form</title>
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>HTML Form</h1>
+  <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post" autocomplete="off">
+    <fieldset class="inputs-block">
+      <legend>Personal information</legend>
+      <p>
+        <label for="surname">Surname:</label>
+        <input type="text" id="surname" name="surname">
+      </p>
+      <p>
+        <label for="name">Name:</label>
+        <input type="text" id="name" name="name">
+      </p>
+      <p>
+        <label for="age">How old are You?</label>
+        <input
+        type="number"
+        name="age"
+        id="age"
+        min="1"
+        max="100"
+        value="12">
+      </p>
```

Comment:
```suggestion
        <input
          type="number"
          name="age"
          id="age"
          min="1"
          max="100"
          value="12"
        >
      </p>
```
1. attributes have the same indentations as nested tags
2. closing bracket `>` should be placed on the new line and have the same indentations as the opening one